packages/components/pie-switch/test/component/pie-switch.spec.ts

```diff
@@ -278,4 +278,107 @@ test.describe('Component: `Pie switch`', () => {
             });
         });
     });
+
+    test.describe('Form integrations', () => {
+        test('should be included in the submitted form data', async ({ page }) => {
+            // Arrange
+            const switchName = 'switch';
+            const switchValue = 'someValue';
+
+            await page.evaluate(() => {
+                const formHTML = `
+                <form id="testForm" action="/default-endpoint" method="POST">
+                    <pie-switch id="pieSwitch" name="switch" value="someValue" label="Click me"></pie-switch>
+                    <button id="submitButton" type="submit">Submit</button>
+                </form>
+                `;
+                document.body.innerHTML = formHTML;
+            });
+
+            // Set up the form submission listener
+            await page.evaluate(() => {
+                const form = document.querySelector('#testForm') as HTMLFormElement;
+
+                form.addEventListener('submit', (e) => {
+                    e.preventDefault(); // Prevent the actual submission
+
+                    // Serialize form data
+                    const formData = new FormData(form);
+                    const formDataObj = {};
+
+                    // This is JS - we don't need TS inside these calls
+                    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+                    // @ts-ignore
+                    formData.forEach((value, key) => { formDataObj[key] = value; });
```

Comment:
This probably is correct but it caught my eye and I thought I'd ask, are value and key the right way round?
```suggestion
                    formData.forEach((value, key) => { formDataObj[key] = value; });
```

Reply:
It does appear to be correct, strangely!